_episodes/04-faceting-and-filtering.md

```diff
@@ -15,27 +15,27 @@ objectives:
 keypoints:
 - "You can use facets and filters to explore your data"
 - "You can use facets and filters work with a subset of data in OpenRefine"
-- "You can easily correct common data issues from a Facet"
+- "You can correct common data issues from a Facet"
 ---
 
 ## Facets
 Facets are one of the most useful features of OpenRefine and can help in both getting an overview of the data and to improve the consistency of the data.
 
 A 'Facet' groups all the values that appear in a column, and then allows you to filter the data by these values and edit values across many records at the same time.
 
-The simplest type of Facet is called a 'Text facet'. This simply groups all the text values in a column and lists each value with the number of records it appears in. The facet information always appears in the left hand panel in the OpenRefine interface.
+The simplest type of Facet is called a 'Text facet'. This groups all the text values in a column and lists each value with the number of records it appears in. The facet information always appears in the left hand panel in the OpenRefine interface.
 
 To create a Text Facet for a column, click on the drop down menu at the top of the publisher column and choose `Facet -> Text Facet`. The facet will then appear in the left hand panel.
 
 The facet consists of a list of values used in the data. You can filter the data displayed by clicking on one of these headings.
 
 You can include multiple values from the facet in a filter at one time by using the `Include` option which appears when you put your mouse over a value in the Facet.
 
-You can also `invert` the filter to show all records which do not match your selected values. This option appears at the top of the Facet panel when you select a value from the facet to apply as a filter.
+You can also `invert` the facet to show all records which do not match your selected values. This option appears at the top of the Facet panel when you select a value from the facet to apply as a filter.
```

Comment:
That is exactly the kind of comments I was hoping for – thanks! :)
Since we're trying to explain the concepts of facets and (text) filters and their functionality, I would like to use more precise words. I found that people are familiar with filtering through facets to narrow down a list (of web shop search results, for example), but that the term *facet* was new to them. Facet is not defined in the episode to be the list of values only. I see the facet as the whole panel.

~To work around this, perhaps we could say that *invert* applies to the *selection*. I think that aligns with some other applications that allow multi-selection: select one or more items, invert the selection, see that your original selections are now the only items not selected.~ I stand corrected – the invert link does not invert the selection as I thought it did; the facet (panel) top gets a different colour and value counts become negative.
I will revert this change.

Reply:
I completely agree we could do a better job of defining terms here. I'm really going off how *I* think about facets and filters, but I think this differentiation is useful - especially in the context of how invert works